scripts/generator/graphql_generator/csharp/SDK/iOS/iOSWebCheckout.cs.erb

```diff
@@ -36,12 +51,46 @@ namespace <%= namespace %>.SDK.iOS {
             if (message.Content == "loadedThankYou") {
                 // Do a round trip to the server to see if the purchase is solid.
                 // Invoke callbacks on cart to pass to Unity
-                OnSuccess();
+                DidCheckoutCompleteSuccessfully(CurrentCheckout.id(), (didSucceed, error) => {
+                    if (didSucceed) {
+                        OnSuccess();
+                        return;
+                    } 
+
+                    if (error != null) {
+                        OnFailure(error);
```

Comment:
Not exactly sure if we should be sending the `HTTP` error to the implementor as an `OnFailure` since the transaction could of be completed but our verification failed to complete. Maybe this should be a `WebCheckoutVerificationError` as well. Thoughts on this @mikkoh?